### ISSUE TITLE
I run the code in Python2, but the loss is NAN

### ISSUE THREAD

**Issue by ghost** [balancap/SSD-Tensorflow#246]

**Comment by suyali** [comment]
You should reduce learning_rate

**Comment by GongliDuan** [comment]
I meet the same problem, and I already set learning rate at very small value.
